applications/asset_tracker/src/cloud_codec/cloud_codec.c

```diff
@@ -721,6 +721,8 @@ int cloud_decode_command(char const *input)
 
 int cloud_decode_init(cloud_cmd_cb_t cb)
 {
+	cJSON_Init();
```

Comment:
Note: CONFIG_CJSON_LIB is selected by CONFIG_NRF_CLOUD if CONFIG_NRF_CLOUD is not set then the user must enalbe CONFIG_CJSON_LIB manually in configs

Reply:
I'll add `CONFIG_CJSON_LIB` to the project configs in a commit to this PR